<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>621</width>
    <height>586</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>虚拟内存管理</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>0</y>
      <width>60</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>页面状态</string>
    </property>
   </widget>
   <widget class="QTableWidget" name="history">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>170</y>
      <width>221</width>
      <height>381</height>
     </rect>
    </property>
    <property name="columnCount">
     <number>2</number>
    </property>
    <column/>
    <column/>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>0</y>
      <width>221</width>
      <height>153</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QLabel" name="label">
         <property name="text">
          <string>物理存储数</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QSpinBox" name="pageNum">
         <property name="minimum">
          <number>1</number>
         </property>
         <property name="maximum">
          <number>6</number>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="setPageNum">
         <property name="text">
          <string>设置</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_4">
       <item>
        <widget class="QRadioButton" name="FIFO">
         <property name="text">
          <string>FIFO</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QRadioButton" name="LRU">
         <property name="text">
          <string>LRU</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="QLabel" name="label_2">
         <property name="text">
          <string>访问逻辑页:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="wantedPage"/>
       </item>
       <item>
        <widget class="QPushButton" name="requestPage">
         <property name="text">
          <string>访问</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_3">
       <item>
        <widget class="QLabel" name="requestNum">
         <property name="text">
          <string>已访问次数:0</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="resetHistory">
         <property name="text">
          <string>重设访问记录</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
   <widget class="QGraphicsView" name="graphicsView">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>361</width>
      <height>531</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>621</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
